README.md

```diff
@@ -61,86 +61,30 @@ The official home of skills for the Mycroft ecosystem.  These skills are written
 | [openHAB](https://github.com/openhab/openhab-mycroft)							| Add an AI Voice assistant to your openHAB system<br>```"turn on Diningroom Light"``` ```"regulate Main Thermostat to 20 degrees"```  |
 
 
-## How to Submit a Skill
+## How to Create and Submit a Skill to the `mycroft-skills` repo
```

Comment:
The TOC at the top of the page is quite outdated now and needs updating or maybe it can be removed completely

Reply:
I will make the edits then run `markdown-toc` on the markdown file to regenerate the ToC. Well spotted!